cmd/simple-dataplane/simple-dataplane.go

```diff
@@ -0,0 +1,341 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"net"
+	"os"
+	"reflect"
+	"runtime"
+	"strings"
+	"sync"
+
+	"github.com/vishvananda/netns"
+
+	"github.com/vishvananda/netlink"
+
+	"github.com/ligato/networkservicemesh/pkg/nsm/apis/simpledataplane"
+	"github.com/ligato/networkservicemesh/pkg/tools"
+	"github.com/sirupsen/logrus"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+const (
+	dataplaneSocket = "/var/lib/networkservicemesh/dataplane.sock"
+)
+
+var (
+	dataplane  = flag.String("dataplane-socket", dataplaneSocket, "Location of the dataplane gRPC socket")
+	kubeconfig = flag.String("kubeconfig", "", "Absolute path to the kubeconfig file. Either this or master needs to be set if the provisioner is being run out of cluster.")
+)
+
+// DataplaneController keeps k8s client and gRPC server
+type DataplaneController struct {
```

Comment:
Is this an internal structure? If so, it should be `dataplaneController` instead of `DataplaneController`.

Reply:
It will become an interface common to all dataplane plugins, so it should be public.

Reply:
Cool, thanks for clarifying!